[{"repo_name": "sass/libsass", "CVE_ID": "CVE-2018-19839", "Problem_Type": "CWE-125", "Description": "In LibSass prior to 3.5.5, the function handle_error in sass_context.cpp allows attackers to cause a denial-of-service resulting from a heap-based buffer over-read via a crafted sass file.", "URL": "https://github.com/sass/libsass/pull/2767", "Tag": "['Patch', 'Third Party Advisory']", "fix_commit_hash": "2cabd116b95d67dcd9d44fcb936dee03f4fc71b9", "inducing_commit_hash_pyszz": ["26df96f095ab78cdf6dc142189950d1b6d67d4a5"]}]